### DISCUSSION TITLE
Sensore sole + pluviometro

### DISCUSSION THREAD

**Discussion by manuelcoders** [indomus/forum#3030] [Hardware & Firmware]
Ciao,
da poco ho smartizzato le mie tapparelle elettriche con degli shelly 2PM ed è tutto perfetto.

Adesso vorrei acquistare dei sensori per rilevare la luce solare, tipo (https://shop.somfy.it/sensore-di-luminosita-senza-fili-radio-rts-sunis.html) e un pluviometro. Con questi due sensori poi vorrei fare della automazioni, ovviamente tutto tramite HA.

Conoscete sensori di questo tipo compatibili con HA? Per il pluviometro ho solo visto la centrale meteo netatmo, per quello relativo al sole non ho trovato nulla! grazie mille

**Comment by calcioscacchi** [comment]
non ti basta il pluviometro e l'entità tramonto nativamente presente su HA?

**Reply by manuelcoders** [reply]
No perché vorrei una cosa precisa ed in base a quei valori muovere le tapparelle.. vorrei magari alle prime gocce d'acqua far scendere le tapparelle, o guardare bene il sole per alzarle e abbassarle

**Comment by calcioscacchi** [comment]
> No perché vorrei una cosa precisa ed in base a quei valori muovere le tapparelle.. vorrei magari alle prime gocce d'acqua far scendere le tapparelle, o guardare bene il sole per alzarle e abbassarle

Ok ,ma non ti conviene procedere per gradi? Mi sembra evidente che il pluviometro serve, non hai altro modo per gestire automazioni, e quello ok.

In un primo step proverei a ragionare col sensore di alba/tramonto, magari giocando con un offset in ore. Non ti costa nulla e magari scopri che ti basta quello. Tanto, che ci perdi? Se ti va bene hai risparmiato una spesa, e se ti va male, la spesa la fai più tardi.

**Reply by manuelcoders** [reply]
ma scusa l'ignoranza, ma come faccio senza avere un sensore di intensità solare a sapere quanto solo batte sulle tapparelle? Il sole invernale è diverso dal sole estivo, e magari un giorno in estate c'è nuvoloso... grazie!

**Reply by calcioscacchi** [reply]
ah pensavo che ti interessasse solamente sapere la quantità di luce intesa come tramonto e alba. Se vuoi regolarlo in base all'intensità ok, secondo me sforzi tantissimo le componenti elettriche, immagino che a ogni nuvola che passa sopra partirà il comando per abbassare

**Reply by manuelcoders** [reply]
eh li poi dovrò gestire bene le automazioni, per evitare di tirare su e giù continuamente le tapparelle ma movimentarle molto poche volte, magari anche a percentuali diverse. Devo provare a capire con funzionano le somfy, loro sono molto avanti su queste cose da quello che ho visto, purtroppo sensori sul mercato però non ne sto trovando! :(
